.github/workflows/unit-tests-jdk-14.yml

```diff
@@ -11,21 +12,24 @@ jobs:
     timeout-minutes: 12
     strategy:
       matrix:
-        java-version: ['14']
-
+        java-version: ["14"]
+    permissions:
+      contents: read
     steps:
-    - uses: actions/checkout@v4
```

Comment:
🗣️ Changes unrelated to hashing steps are due to **indentation**! No change to testing or functionalities otherwise.